source/partials/_changelog.erb

```diff
@@ -4,10 +4,11 @@
 
 | Publication date | Update                                                                                                                                                                                                                                                                                                                                           |
 |------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Jun 21 2024       | Updated guidance <a href="/integrate-with-integration-environment/authenticate-your-user/#error-handling-for-make-a-request-to-the-authorize-endpoint">"Error handling for ‘Make a request to the /authorize endpoint"</a> to clarify the <code>{"message": "Internal server error"}</code>HTTP 502 Bad gateway error.                                                                                                   |
-| Jun 18 2024       | Includes example data to help with building mocks: "<a href="/test-your-integration/build-mocks/#access-example-data">Access example data</a>".                                                                                                   |
-| May 22 2024       | New guidance "<a href="/test-your-integration/using-integration-for-testing/">Using the integration environment for end-to-end testing</a>" to explain how to use the integration environment for end-to-end testing.                                                                                                   |
-| May 17 2024       | New guidance "<a href="/test-your-integration/build-mocks/#build-mocks-to-work-with-gov-uk-one-login">Build mocks to work with GOV.UK One Login</a>" to explain how to build mocks as a part of testing your service.                                                                                                   |
+| Jul 3 2024       | Remove the `https://vocab.account.gov.uk/v1/socialSecurityRecord` claim                                                                                                     |
+| Jun 21 2024      | Updated guidance <a href="/integrate-with-integration-environment/authenticate-your-user/#error-handling-for-make-a-request-to-the-authorize-endpoint">"Error handling for ‘Make a request to the /authorize endpoint"</a> to clarify the <code>{"message": "Internal server error"}</code>HTTP 502 Bad gateway error.                                                                                                   |
```

Comment:
we're mixing tenses here ... should it be past or present?

Reply:
good point let me bring it in line with the rest of the changelog

Reply:
I suspect the changelog needs looking at as a whole, I merely mimicked the earlier wording